domain-check.bash

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+# This is meant to be an interim script to (1) evaluate we have
+# enough capabilities inside of the urlgetter pseudo-test and (2)
+# prototype the next generation Web Connectivity.
+#
+# Very alpha. Very basic. Needs much more love.
+#
+# Inspired by great chats and great code. Main source of ispiration is
+# currently https://github.com/Jigsaw-Code/net-analysis/blob/master/netanalysis/blocktest/measure.sh
+
+function fatal() {
+  echo "FATAL: $@" 1>&2
+  exit 1
+}
+
+if [ $# -ne 1 ]; then
```

Comment:
Prefer `[[]]` to `[]`: https://google.github.io/styleguide/shellguide.html#s6.3-tests

BTW, that's a pretty good style guide. You may want to use it if you don't already have one.

Reply:
Thanks for letting me know about this style guide, I was not aware of that!